doc/user_guide/bank/essentials_calendar.md

```diff
@@ -1 +1,34 @@
-Not yet written (We have someone working on it though as of 20240719)
\ No newline at end of file
+# Pick Ups & Deliveries Calendar
+
+Pick Ups & Deliveries shows scheduled and completed distributions in a calendar format on a monthly basis.
+
+Click on "Pick Ups & Deliveries" in the left-hand menu to view the calendar.
+
+![PickUps & Delivery Calendar](images/pickup&delivery.jpg)
+
+Click on any scheduled distribution to view more details about it.
```

Comment:
Somehow my note around this one didn't take -- this isn't actually providing details for one distribution,  but for all the distributions for that day.    Please rephrase this to take that into account.

Reply:
Looks like I missed this.